test/sprinkler_web/mqtt/handler_test.exs

```diff
@@ -1,6 +1,8 @@
 defmodule SprinklerWeb.Mqtt.HandlerTest do
   use ExUnit.Case, async: true
 
+  @telemetry "telemetry"
```

Comment:
Why do you need this?

Reply:
used as topic to subscribe to, on last test